tests.js

```diff
@@ -1223,6 +1223,7 @@ window.Specs = {
 	"mediaqueries-5": {
 		"title": "Media Queries Level 5",
 		"links": {
+			"tr": "mediaqueries-5",
```

Comment:
Currently not existing

Reply:
It's there, though :thinking: : https://www.w3.org/TR/mediaqueries-5/

Reply:
Ho sorry.